eventhubs/client/README.md

```diff
@@ -0,0 +1,192 @@
+  aggregation, complex computation, and filtering. Processing may also involve distribution or storage of the information
+  in a raw or transformed fashion. Event Hub consumers are often robust and high-scale platform infrastructure parts 
+  with built-in analytics capabilities, like Azure Stream Analytics, Apache Spark, or Apache Storm.
+
+- A **partition** is an ordered sequence of events that is held in an Event Hub. Partitions are a means of data 
+  organization associated with the parallelism required by event consumers. Azure Event Hubs provides message streaming
+  through a partitioned consumer pattern in which each consumer only reads a specific subset, or partition, of the 
+  message stream. As newer events arrive, they are added to the end of this sequence. The number of partitions is
+  specified at the time an Event Hub is created and cannot be changed.
+
+- A **consumer group** is a view of an entire Event Hub. Consumer groups enable multiple consuming applications to each 
+  have a separate view of the event stream, and to read the stream independently at their own pace and from their own
+  position. There can be at most 5 concurrent readers on a partition per consumer group; however it is recommended that 
+  there is only one active consumer for a given partition and consumer group pairing. Each active reader receives all of
+  the events from its partition; if there are multiple readers on the same partition, then they will receive duplicate 
+  events. 
+
+For more concepts and deeper discussion, see: [Event Hubs Features][event_hubs_features]. Also, the concepts for AMQP 
+are well documented in [OASIS Advanced Messaging Queuing Protocol (AMQP) Version 1.0][oasis_amqp_v1].
+
+## Examples
+will re-doc this section tonight
+- [Inspect Event Hub and partition properties][sample_get_event_hubs_metadata]
+- [Publish an event to an Event Hub][sample_send_event]
+- [Consume events from an Event Hub partition][sample_receive_event]
+
+## Troubleshooting
+
+### Enable client logging
+You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For example,
+setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can be found here: [log levels][log_levels].
+
+### Enable AMQP transport logging
+If enabling client logging is not enough to diagnose your issues. You can enable logging to a file 
+in the underlying AMQP library, [Qpid Proton-J][qpid_proton_j_apache]. Qpid Proton-J uses `java.util.logging`. You can enable logging by create a configuration file with the contents below. Or set `proton.trace.level=ALL` and whichever configuration options you want for the `java.util.logging.Handler` implementation. Implementation classes and their options can be found in [Java 8 SDK javadoc][java_8_sdk_javadocs].
+
+#### Sample "logging.config" file ####
+``` logging config
```

Comment:
logging config is not a language that is recognised by their linguist file. https://help.github.com/en/articles/creating-and-highlighting-code-blocks#syntax-highlighting